test/test_sync_handlers.js

```diff
@@ -34,7 +32,7 @@ describe('$fh.sync', function () {
       init: sinon.spy(),
       createDataset: sinon.spy(),
       stopDatasetSync: sinon.spy(),
-      stopAllDatasetSync: sinon.spy(),
+      stopAllDatasetSync: sinon.stub().callsArgAsync(0),
```

Comment:
@evanshortiss this change ensures the global `after` will have its callback called when it does `fh.sync.stopAll`